src/content/structured/get-started/figma.mdx

```diff
@@ -1,57 +1,58 @@
 ---
 path: "/get-started/figma"
-
+ 
 navPriority: 5
-
-date: "2022-11-15"
-
+ 
+date: "2023-09-07"
+ 
 title: "Get the Figma kit"
-
-subTitle: "Reusable and flexible Figma components used to design, prototype and hand-off designs for apps and services."
-
+ 
+subTitle: "Reusable and flexible Figma components used to design, prototype and hand-off designs for apps and digital products."
+ 
 contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/figma.mdx"
 ---
-
-import { IcAlert } from "@ukic/react";
+ 
 import figmaFig1 from "./images/assets2x.png";
 import figmaFig2 from "./images/libraries2x.png";
-
+ 
 ## About the Figma UI Kit
-
+ 
 We've built a Figma UI Kit to use alongside the guidance and code you'll find on this website. It'll make it easier to prototype and build things quickly.
-
-All Figma resources have been built with our users in mind, with a strong focus on making sure accessibility is embedded from the start. That way what we can provide an amazing experience, usable for everyone
-
-## Accessing the library
-
-<IcAlert
-  title="Need to know"
-  message="The Figma UI Kit is currently only available to internal users, but we will be sharing it publicly soon."
-  variant="info"
-/>
-
-The UI Kit is turned on by default for all projects within our Figma organisation.
-
+ 
+All Figma resources have been built with our users in mind, with a strong focus on making sure accessibility is embedded from the start. That way what we can provide an amazing experience, usable for everyone.
+ 
+## Accessing the library for internal users
+ 
+The UI Kit is turned on by default for all projects within our community. Check that you are using the latest version.
```

Comment:
```suggestion
The UI Kit is turned on by default for all projects within our community. Ensure that you are using the latest version.
```

Reply:
Going to leave as is so that we follow our 'use simple words' guidance. :)

https://design.sis.gov.uk/styles/content-style/voice-and-tone#swap-formal-words-for-normal-ones